src/ui/components/ShowMoreCard/index.js

```diff
@@ -40,7 +40,7 @@ export class ShowMoreCardBase extends React.Component {
   truncateToMaxHeight = (contents) => {
     // If the contents are short enough they don't need a "show more" link; the
     // contents are expanded by default.
-    if (contents.clientHeight > MAX_HEIGHT) {
+    if (contents.clientHeight >= MAX_HEIGHT) {
```

Comment:
Need this as, when content loads the due to max height the height of content equals 150px, but again on `componentWillReceiveProps`, `truncateToMaxHeight` is invoked so even at 150px we need to set `{ expanded: false }`